src/UnitTest/test_quote_prov.cpp

```diff
@@ -965,11 +942,11 @@ void SetupEnvironment(std::string version)
         setenv("AZDCAP_COLLATERAL_VERSION", version.c_str(), 1);
         if (version == "v3")
         {
-            setenv(
-                "AZDCAP_BASE_CERT_URL",
-                "https://americas.test.acccache.azure.net/sgx/certificates",
-                1);
-        }
+			setenv(
+				"AZDCAP_BASE_CERT_URL",
+                "https://dev.test.acccache.azure.net/sgx/certificates",
```

Comment:
We may not need this entire if block if we're no longer needing to test against dev clusters and instead can run the tests against prod itself. The v3 block was put in because prod THIM didn't have the prod icelake enabled as intel didn't have it enabled. Suggestion to just default to prod in general again like I believe it used to be.

Reply:
Removed